app/models/manageiq/providers/ibm_power_hmc/infra_manager/host.rb

```diff
@@ -1,6 +1,8 @@
 class ManageIQ::Providers::IbmPowerHmc::InfraManager::Host < ::Host
   include ManageIQ::Providers::IbmPowerHmc::InfraManager::MetricsCaptureMixin
 
+  supports :power_off
```

Comment:
@agrare I don't know why I have this error in the logs but I wonder if it's not because I'm using the wrong keyword here?

Reply:
Hey @CallisteH we have to have `supports_not :feature_name` in the base class (in this case `::Host`) so that we can ask `instance.supports?(:feature_name)` on an instance of any class